config/install/user.role.approver_plus.yml

```diff
@@ -59,16 +59,7 @@ permissions:
   - 'create image media'
   - 'create paragraph library item'
   - 'create url aliases'
-  - 'delete any audio media'
-  - 'delete any document media'
-  - 'delete any embedded_video media'
-  - 'delete any file media'
-  - 'delete any image media'
-  - 'delete any media'
   - 'delete media'
-  - 'delete own audio media'
-  - 'delete own document media'
-  - 'delete own image media'
```

Comment:
same no need to remove all these, as the ticket says only to remove node permissions